src/entities/movie/api/index.ts

```diff
@@ -1,11 +0,0 @@
-import { TrendingMovie } from "../model/types";
-
-const BASE_URL = 'http://localhost:3001';
-
-export const getTrendingMovies = async (): Promise<TrendingMovie[]> => {
```

Comment:
Let's keep this method
So we'll have 3 parts of the data manipulating on FE
1. (this method) Part for getting data from network. Here you can catch network errors, use needed headers for your reqeuse/response, so do anything you need for network communication And this part returns DTO 
2. (hook) Part which uses previous part and transform data from network for our needs as we want DTO to DO
3. (component/other hook/so any ui part let's say) Part for view data, data is already transformed and we can get it and render